### DISCUSSION TITLE
frankenphp & davis -> worker restarts every ms

### DISCUSSION THREAD

**Discussion by extrafein** [php/frankenphp#473] [Q&A]
Is here anyone who can give me a hint what I can do to get this nice project: [davis](https://github.com/tchapi/davis) runniing with the worker mode of frankenphp?

> davis: 🗓 A simple, fully translatable admin interface for sabre/dav based on Symfony 5 and Bootstrap 5, initially inspired by Baïkal.

After starting the docker-container the worker is going mad and is restarting all the time; eating up all cpu ressources.

Thank you in advance for your support.

This is my Caddyfile. Nothing realy special:
```
{
        http_port 80

        frankenphp {
               worker /app/public/index.php
                auto_https off
        }
        auto_https off
        order php_server before file_server
}

localhost :80 {

        encode zstd gzip

        root * /app/public

        redir /.well-known/carddav /dav 301
        redir /.well-known/caldav /dav 301

        @canonicalPath {
                file {path}/index.php
                not path */
        }
        redir @canonicalPath {path}/ 308

        @indexFiles file {
                try_files {path} {path}/index.php index.php
                split_path .php
        }
        rewrite @indexFiles {http.matchers.file.relative}

        # FrankenPHP!
        php_server {
                resolve_root_symlink
        }
        file_server
}
```

**Comment by withinboredom** [comment]
It seems like maybe the code doesn't support worker mode? If the code exits, or terminates, then the worker restarts.
